From d69c1ff524f7c22a5d215e3c99ce2f81204dd0ab Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@lanl.gov>
Date: Mon, 21 Nov 2022 10:01:23 -0700
Subject: [PATCH] update

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6ddf3d4..756de37 100644
--- a/setup.py
+++ b/setup.py
@@ -38,5 +38,6 @@
         'matplotlib>=3.5.3',
         'numpy>=1.23.2',
         'pandas>=1.4.3',
-    ]
+    ],
+    cmdclass=versioneer.get_cmdclass(),
 )
